pandas/core/indexes/period.py

```diff
@@ -823,6 +823,9 @@ def memory_usage(self, deep=False):
             result += self._int64index.memory_usage(deep=deep)
         return result
 
+    def _convert_list_indexer(self, keyarr, kind=None):
+        return super().get_indexer_for(keyarr)
+
```

Comment:
would this be incorrect on TimedeltaIndex?  potentially move to mixin class?

Can you add  a comment pointing back to the relevant issue, or why this is calling super for a different method

Reply:
does `kind` not matter?